pkgs/data/fonts/scientifica/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
```

Comment:
Fonts are usually no derivation but just a download with a post phase. You should find some other fonts that do it like this.